src/actions/account.js

```diff
@@ -190,7 +192,15 @@ export const updateAccountDelegateStats = account =>
     }));
   };
 
-// eslint-disable-next-line max-statements
+/**
+ * This action is used on login to fetch account info for all enabled token
+ *
+ * @param {Object} data - for hardware wallets it contains publicKey and hwInfo,
+ *    otherwise contains passphrase
+ * @param {String} passphrase - BIP39 passphrase of the account
+ * @param {String} publicKey - Lisk publicKey used for hardware wallet login
+ * @param {Object} hwInfo - info about hardware wallet we're trying to login to
+ */
 export const login = ({ passphrase, publicKey, hwInfo }) => async (dispatch, getState) => {
```

Comment:
@slaweet I checked the comments and all looks good, in the other hand jus as a comment I think this login action can be improve, so don't know if that will be part of this PR or another PR

Reply:
@osvaldovega improving login action can be part of this PR if it will keep the same functionality. What do you have in mind?

One separate improvement of login I'm aware of is https://github.com/LiskHQ/lisk-hub/issues/2197 but, but so far it is missing specification of what exactly should happen with BTC in that case.